### ISSUE TITLE
Image width displayed wrong

### ISSUE THREAD

**Issue by achimwagenknecht** [mb21/panwriter#115]
The image widths don't make sense to me. They are displayed much to small. In addition, sometimes the tag is not compiled but displayed. See screenshot.
![test_001](https://user-images.githubusercontent.com/10030555/188899072-176ef769-40d2-4f74-a5d4-fa083c147ff1.png)

**Comment by Moonbase59** [comment]
`{width=…}` is probably handled as an HTML attribute—as such it would always be in pixels. The generated preview is always in `px` width, regardless of what length unit one specifies.

One exception is using `%`, which seems to work rather well. Using `mm` as an example would only work with CSS property values (and if the output media is known). Using absolute length values isn’t recommended, although I’d like that too.

I’m unsure if this is a Panwriter preview or a Pandoc problem, though.

**Comment by mb21** [comment]
Yes, pretty sure the attribute is currently just passed through as is in the preview, unlike pandoc. This would have to be implemented in https://github.com/mb21/markdown-it-pandoc/
